appendices/VK_MESA_rect_list.txt

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Clément Guérin
```

Comment:
The "(c)" is redundant and not needed, BTW. Just "Copyright <date> <author>" is fine.